text-pango.c

```diff
@@ -111,7 +111,9 @@ void render_text(struct pango_source *src)
 
 	/* Allocate and initialize Cairo surface */
 	int stride = cairo_format_stride_for_width(CAIRO_FORMAT_ARGB32, src->width);
-	surface_data = bzalloc(stride * src->height);
+	/* UGLY HACK: don't bzalloc(0), it causes a crash in newer OBS */
+	size_t allocsize = stride * src->height;
+	surface_data = bzalloc(allocsize != 0 ? allocsize : 1);
```

Comment:
If the texture is 0 sized at this point it should be safe to return here, since nothing is going to be rendered? The texture was already cleared at the top.

Reply:
Ok I'll see if I can do that